packages/panels/docs/02-getting-started.md

```diff
@@ -168,11 +172,11 @@ Forms\Components\TextInput::make('name')
     ->maxLength(255)
 ```
 
-Attempt to submit the form to create a new patient without a name, and observe that a message is displayed informing you that the name field is required.
+Attempt to submit the form to create a new patient without a name and observe that a message is displayed informing you that the name field is required.
 
 #### "Type" select
 
-Let's add a second field, for the type of patient - either a cat, dog or rabbit. Since there are a set of options that should be chosen from, it's best to use a [select](../forms/fields/select) field for this:
+Let's add a second field for the type of patient: a choice between a cat, dog, or rabbit. Since there's a fixed set of options to choose from, a [Select](../forms/fields/select) field works well:
```

Comment:
```suggestion
Let's add a second field for the type of patient: a choice between a cat, dog, or rabbit. Since there's a fixed set of options to choose from, a [select](../forms/fields/select) field works well:
```